x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/index.tsx

```diff
@@ -137,7 +143,7 @@ export const AlertIndex = memo(() => {
         return (
           <Link
```

Comment:
Should be using a connected `EuiLink`. I think @parkiino 's PR will include a common one in our `components/` dir, so you can use that when its merged in.